releases/cloud.md

```diff
@@ -15,7 +15,7 @@ Get future release notes emailed to you:
 {% include marketo.html %}
 
 {{site.data.alerts.callout_version}}
-As of July 6, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v21.2.12](v21.2.html) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.2](v22.1.html).
+As of July 11, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v22.1.1](v22.1.html) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.2](v22.1.html).
```

Comment:
IDK if there is a convention in the Cloud docs, but the links to minor versions could be more specific, e.g. `[v22.1.1](v22.1.html#v22-1-1)`